src/backbone.syphon.js

```diff
@@ -141,13 +140,13 @@ var getElementType = function(el) {
   return type.toLowerCase();
 };
 
-// If a form element is given, just return it.
-// Otherwise, get the form element from the view.
+// If a dom element is given, just return the form fields.
+// Otherwise, get the form fields from the view.
 var getForm = function(viewOrForm) {
-  if (_.isUndefined(viewOrForm.$el) && viewOrForm.tagName.toLowerCase() === 'form') {
-    return viewOrForm;
+  if (_.isUndefined(viewOrForm.$el)) {
+    return $(viewOrForm).children(':input');
   } else {
-    return viewOrForm.$el.is('form') ? viewOrForm.el : viewOrForm.$('form')[0];
+    return viewOrForm.$(':input');
```

Comment:
Yeeeeah, but things seem to be pointing in that direction. Consider also #72. I don't think reducing coupling with jQuery can be reduced until a major release as out-of-the-box matching (or not matching), is only currently provided by overriding the key and value matchers.

The new Syphon would see the matching construct exposed closer to serialise so the choice of jQuery would fall down to the implementor